Modules/_hashopenssl.c

```diff
@@ -522,6 +619,27 @@ get_hashlib_utf8name_by_evp_md(const EVP_MD *md)
     return get_hashlib_utf8name_by_nid(EVP_MD_nid(md));
 }
 
+/*
+ * Return 1 if the FIPS query property should be disabled.
+ *
+ * Note that returning 0 does not necesasrily mean that the
+ * fetched algorithm will be available. For instance, a FIPS
+ */
```

Comment:
```suggestion
/*
 * Return 1 if the property query clause [1] must be "-fips" and 0 otherwise.
 *
 * Note that returning 0 does not necessarily mean that the fetched algorithm
 * is available or usable by EVP_MD_* functions.
 *
 * [1] https://docs.openssl.org/master/man7/property
 */
```